whitesource.config

```diff
@@ -0,0 +1,372 @@
+###############################################################
+# WhiteSource Unified-Agent configuration file
```

Comment:
Can we add links to the WhiteSource documentation and user guide in this file at least?

Reply:
Added some useful document links for this file.